[01/11/2019 11:22:46] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:22:46] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:22:46] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:22:47] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:22:47] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:22:47] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:23:37] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:23:37] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:23:37] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:23:45] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:23:45] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:23:45] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:26:36] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:26:36] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:26:36] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:26:37] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:26:37] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:26:37] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:32:57] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:32:57] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:32:57] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:32:58] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:32:58] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:32:58] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:33:48] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
name 'passwor' is not defined
[01/11/2019 11:34:29] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
name 'passwor' is not defined
[01/11/2019 11:35:34] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:35:34] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:35:34] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:35:34] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:35:34] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:35:34] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:35:40] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
name 'passwor' is not defined
[01/11/2019 11:35:49] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
name 'passwor' is not defined
[01/11/2019 11:37:12] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:37:12] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:37:12] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:37:18] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:37:18] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:37:18] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:37:19] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:37:19] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:37:19] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:39:14] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:39:14] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:39:14] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:39:15] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:39:15] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:39:15] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:39:22] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
name 'passwor' is not defined
[01/11/2019 11:39:26] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
name 'passwor' is not defined
[01/11/2019 11:39:29] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
name 'passwor' is not defined
[01/11/2019 11:49:33] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:49:33] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:49:33] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:49:34] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:49:34] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:49:34] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:49:37] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:49:37] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:49:37] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:49:41] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:49:41] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:49:41] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:49:44] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:49:44] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:49:44] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:49:51] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:49:51] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:49:51] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:49:54] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:49:54] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:49:54] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:54:23] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
name 'passwor' is not defined
[01/11/2019 11:55:10] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 11:55:10] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 11:55:10] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 11:55:56] [ERROR] root [get_jenkins_connection] [jenkins_routes.py | 27] Falha na conexão com o Jenkins: 
HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /crumbIssuer/api/json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd7154e3c88>: Failed to establish a new connection: [Errno 111] Connection refused',))
[01/11/2019 14:51:33] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 14:51:33] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 14:51:33] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 14:52:53] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 14:52:53] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 14:52:53] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
[01/11/2019 14:52:53] [INFO] root [<module>] [app.py | 23] Carregando variáveis de ambiente ...
[01/11/2019 14:52:53] [INFO] root [<module>] [app.py | 26] Configurando a aplicação ...
[01/11/2019 14:52:53] [INFO] root [<module>] [app.py | 41] Iniciando a aplicação ...
